confidential-datahub/kms/src/api.rs

```diff
@@ -68,6 +69,6 @@ pub trait Setter: Send + Sync {
     /// Get secret. Different secret manager will use different parameters inside
     /// `annotations`.
     ///
-    /// The returned [`Annotations`] is the public parameters of the Vault.
-    async fn set_secret(&mut self, name: &str, value: &str) -> Result<(Vec<u8>, Annotations)>;
+    /// The returned [`SetSecretOption`] is the public parameters of the 'set_secret' api.
+    async fn set_secret(&mut self, name: &str, value: &str) -> Result<(Vec<u8>, SetSecretOption)>;
```

Comment:
How about `VaultContext`?

Reply:
The following changes are applied:
- 'EncryptOption' -> 'SealerOption'
- 'DecryptOption' -> 'VaultOption'
I think they could be more accurate.